types/audit-details.d.ts

```diff
@@ -211,6 +211,11 @@ declare global {
         urlProvider: 'network' | 'comment';
         line: number;
         column: number;
+        original?: {
```

Comment:
do we have a standard-ish way of referring to this in the codebase yet? (or is `original` it?)

it would be good to break down the cases in here, as well, because it's not completely clear. e.g. if `urlProvider` is `'comment'`, does that mean `url` is still the name of the bundled js or the name of the (source map?) file in the source map comment?

Reply:
FWIW [mozilla's source map](https://github.com/mozilla/source-map/#sourcemapconsumerprototypeoriginalpositionforgeneratedposition) uses `generated` and `original`.

`source-location` is the only place we store line or column numbers, and this is the first time we're doing anything with the mapped location anywhere, so yeah `original` is new and is what I suggest we go with.

I see `urlProvider = 'comment'` as an optional alias for the network resource. If there's a map, the better name is the mapped file. When there's a mapped file + a `comment` I relegate the `sourceURL=<comment url>` info to the `title` tooltip. I'll add this info as a comment.

Reply:
> I'll add this info as a comment.

bump :) maybe...

```suggestion
        /** The original file location information from the source map. */
        original?: {
```